### PR TITLE
Fix-2120

### LINKED ISSUE #2120: Player Select Menu has mode selection issues
If the player clicks on the mode that is currently selected in the player start screen, then the bottom text, which says the mode, disappears.

### PR DESCRIPTION
**Pull request by FSCyborg**
Move the bottom text reset to inside the if so that it isn't kept empty by accident.

Let me know if I should move the comment around at all, or if I need to make any other changes.

Designed to fix #2120

### CODE REVIEW

**Comment by FSCyborg** [comment]
Just ran the updated build, and this fixes the issue.

File: code/menuui/playermenu.cpp [hunk 1]
```diff
@@ -663,14 +663,17 @@ void player_select_button_pressed(int n)
 		break;
 
 	case SINGLE_BUTTON:
-		player_select_set_bottom_text("");
-
+
```
Comment (asarium) [inline code/menuui/playermenu.cpp]: This empty line should be removed.

File: code/menuui/playermenu.cpp [hunk 2]
```diff
@@ -681,7 +684,6 @@ void player_select_button_pressed(int n)
 		break;
 
 	case MULTI_BUTTON:
-		player_select_set_bottom_text("");
```
Comment (asarium) [inline code/menuui/playermenu.cpp]: Another empty line.

File: code/menuui/playermenu.cpp [hunk 3]
```diff
@@ -663,14 +663,17 @@ void player_select_button_pressed(int n)
...
-
+		
 		Player_select_autoaccept = 0;
 		// switch to single player mode
 		if (Player_select_mode != PLAYER_SELECT_MODE_SINGLE) {
 			// play a little sound
 			gamesnd_play_iface(InterfaceSounds::USER_SELECT);
 
+			// moving this here, from outside the if, keeps bottom text from being empty by accident,
+			// when the player accidentally presses the already-lit mode
```
Comment (asarium) [inline code/menuui/playermenu.cpp]: A tip for writing comments, write them to be stateless to make them easier to understand without looking at the file history. I would write it something like this:
```suggestion
      // Only do this when changing modes, keeps bottom text from being empty by accident,
			// when the player accidentally presses the already-lit mode
```

File: code/menuui/playermenu.cpp [hunk 4]
```diff
@@ -694,6 +696,10 @@ void player_select_button_pressed(int n)
...
```
Comment (asarium) [inline code/menuui/playermenu.cpp]: See above.

**Comment by FSCyborg** [comment]
I should have just fixed everything you addressed.

**Pull request merged**